src/components/BulkEnrollmentPage/index.test.jsx

```diff
@@ -1,94 +1,138 @@
     loading: false,
+    count: 2,
   },
 });
-
-LicenseManagerApiService.fetchSubscriptions.mockReturnValue(subscriptions);
+const singleSubscription = Promise.resolve({
+  data: {
+    results: [sub1],
+    errors: null,
+    setErrors: jest.fn(),
+    forceRefresh: jest.fn(),
+    loading: false,
+    count: 1,
+  },
+});
+const mockSingleSubscriptionCount = Promise.resolve({
+  data: [
+    { status: 'unassigned', count: 0 },
+    { status: 'assigned', count: 0 },
+    { status: 'activated', count: 1 }],
+});
 
 const fakeStore = {
   portalConfiguration: {
     enterpriseId: 'foo',
+    enableCodeManagementScreen: false,
+    enableSubscriptionManagementScreen: true,
   },
 };
 const mockStore = configureMockStore([thunk]);
 const BulkEnrollmentWrapper = () => (
-  <MemoryRouter>
-    <Provider store={mockStore(fakeStore)}>
-      <BulkEnrollmentPage />
-    </Provider>
-  </MemoryRouter>
+  <Provider store={mockStore(fakeStore)}>
+    <BulkEnrollmentPage />
+  </Provider>
 );
 
-describe('<BulkEnrollmentPage />', () => {
+LicenseManagerApiService.fetchSubscriptions.mockImplementation(() => subscriptions);
+LicenseManagerApiService.fetchSubscriptionUsersOverview.mockImplementation(() => mockSubscriptionCount);
+
+describe('<BulkEnrollmentPage /> Multiple Subscriptions', () => {
   it('renders the bulk enrollment component', async () => {
     renderWithRouter(<BulkEnrollmentWrapper />);
     screen.getByText('Subscription Enrollment');
     await act(() => subscriptions);
   });
-  it.skip('renders the subscription picker', async () => {
-    renderWithRouter(<BulkEnrollmentWrapper />, { route: `/sluggo/admin/${ROUTE_NAMES.bulkEnrollment}` });
+  it('renders the subscription picker', async () => {
+    renderWithRouter(<BulkEnrollmentWrapper />, { route: `/${testSlug}/admin/${ROUTE_NAMES.bulkEnrollment}` });
     expect(await screen.findByText('Cohorts')).toBeInTheDocument();
     expect(await screen.findByText(sub1.title)).toBeInTheDocument();
     expect(await screen.findByText(sub2.title)).toBeInTheDocument();
-
+  });
+  it('renders the course search page', async () => {
+    renderWithRouter(<BulkEnrollmentWrapper />, { route: `/${testSlug}/admin/${ROUTE_NAMES.bulkEnrollment}/${sub1.uuid}` });
+    expect(await screen.findAllByText(ADD_COURSES_TITLE)).toHaveLength(2);
+  });
+  it('renders course search page when navigating via subscription picker', async () => {
+    renderWithRouter(<BulkEnrollmentWrapper />, { route: `/${testSlug}/admin/${ROUTE_NAMES.bulkEnrollment}` });
     await act(() => subscriptions);
+    // The expiration modal makes the enroll learners link inaccessible, so we need to dismiss it first.
+    const modalButton = await screen.getByRole('button', { name: 'OK' });
+    userEvent.click(modalButton);
+
+    const sub1Button = screen.getAllByRole('link', { name: 'Enroll learners' })[0];
+    userEvent.click(sub1Button);
+    expect(screen.getByRole('heading', { level: 2 })).toHaveTextContent(ADD_COURSES_TITLE);
```

Comment:
It appears once in the list of steps at the top and once as a header. I think this is better, tbh. It's possible that the title and the name of the step won't stay the same forever. This will break if we change the header level, but I think that's unlikely and kind of merits a test breaking bc accessibility.